charts/mirrorbits/tests/custom_values_test.yaml

```diff
@@ -28,6 +48,7 @@ tests:
   - it: Should set the correct service selector labels when a nameOverride is specified
     template: service.files.yaml
     set:
+      # Overides the fullname overide \o/
```

Comment:
```suggestion
      # Overrides the name override \o/
```
nameOverride != fullnameOverride

Reply:
if both are specified, then `nameOveride` wins (hence the `fullnameOverride` set by default on the top level custom value)

Reply:
Changed as per your review feedbacks